src/components/clusterDetail/ClusterInstallationProgressCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Card, Title, CardBody, CardHeader, CardExpandableContent } from '@patternfly/react-core';
+import { Cluster, Credentials, getClusterCredentials, stringToJSON } from '../../api';
+import { AlertsContextProvider } from '../AlertsContextProvider';
+import CancelInstallationModal from './CancelInstallationModal';
+import ClusterCredentials from './ClusterCredentials';
+import ClusterInstallationError from './ClusterInstallationError';
+import ClusterProgress from './ClusterProgress';
+import FailedHostsWarning from './FailedHostsWarning';
+import ResetClusterModal from './ResetClusterModal';
+import HostsTable from '../hosts/HostsTable';
+import ClusterDetailsButtonGroup from './ClusterDetailsButtonGroup';
+import { ClusterStatusIcon } from '../clusters/ClusterStatus';
+
+type AIClusterDetailsCardProps = {
+  clusterStringObj: string;
+};
+
+const getID = (suffix: string) => `cluster-detail-${suffix}`;
+
+const AIClusterDetailsCard: React.FC<AIClusterDetailsCardProps> = ({ clusterStringObj }) => {
+  const cluster: Cluster = stringToJSON<Cluster>(clusterStringObj) as Cluster;
+
+  const [cancelInstallationModalOpen, setCancelInstallationModalOpen] = React.useState(false);
+  const [resetClusterModalOpen, setResetClusterModalOpen] = React.useState(false);
+  const [credentials, setCredentials] = React.useState<Credentials>();
+  const [credentialsError, setCredentialsError] = React.useState();
+  const [isCardExpanded, setIsCardExpanded] = React.useState(cluster.status !== 'installed');
+
+  const fetchCredentials = React.useCallback(() => {
```

Comment:
It would be good to better consolidate the `ClusterInstallationProgressCard` and `ClusterDetail` components as there is duplicated code and the components have diverged since addition of monitored operators.